### PR TITLE
fix an exception in the msteams sink

### PR DESCRIPTION
**Pull request by aantn**
Note that there are a number of changes due to autoformatting, but only a single line was actually changed. I'll point out which line in the diff.

### CODE REVIEW

File: src/robusta/integrations/msteams/msteams_elements/msteams_text_block.py [hunk 1]
```diff
@@ -3,34 +3,46 @@
...
+                is_visible,
+                separator,
+                font_size,
+                horizontal_alignment,
+            )
+        )
 
     def __to_msteams_text(self, text: str) -> str:
-        teams_text = MsTeamsMarkDownFixUrl().fix_text(text)
+        teams_text = MsTeamsMarkDownFixUrl().fix_text(str(text))
```
Comment (aantn) [inline src/robusta/integrations/msteams/msteams_elements/msteams_text_block.py:31]: This is the actual fix. Sometimes text was a number and it caused an exception down the call stack.

File: src/robusta/integrations/msteams/msteams_mark_down_fix_url.py [hunk 2]
```diff
@@ -1,26 +1,23 @@
-
+import logging
```
Comment (arikalon1) [inline src/robusta/integrations/msteams/msteams_mark_down_fix_url.py:1]: Why did you add it?

**Review by arikalon1 — APPROVED** [review]
Looks good.
Left 1 minor comment

File: src/robusta/integrations/msteams/msteams_mark_down_fix_url.py (hunk 2 above)
Comment (aantn) [reply src/robusta/integrations/msteams/msteams_mark_down_fix_url.py]: Whoops, fixed it. (It was from an earlier version when I was debugging the issue)
